vllm/model_executor/layers/mamba/ops/ssd_bmm.py

```diff
@@ -0,0 +1,267 @@
+# Copyright (c) 2024, Tri Dao, Albert Gu.
+# Adapted from https://github.com/state-spaces/mamba/blob/main/mamba_ssm/ops/triton/ssd_bmm.py
+
+# ruff: noqa: E501,SIM102
+"""We want triton==2.1.0 or 2.2.0 for this
+"""
```

Comment:
I think this comment is out-of-date and should be deleted (Likely triton will be >= 3.1.0)

ditto for other files

Reply:
Done in https://github.com/vllm-project/vllm/pull/10909/commits/d788db694330d27ffc0f269dfbb1ccef3eb82f72, also removed softplus.py and reused the one already in mamba_ssm.py